plugins/minify.js

```diff
@@ -206,6 +208,7 @@
         callback(minified.substr(header.length, minified.length-(header.length+footer.length+1)));
       }
     }
+    // End of unused block
```

Comment:
This whole block appears potentially unused, the `minifyCallback` is defined and then conditionally set to a different callback function, but it's never used anywhere

Reply:
Wow, thanks for this spot - I'm pretty sure that in `minifyCodeGoogle/minifyCodeEsprima` the idea was that `code => minifyCode` and so on